c-pallets/segment-book/src/benchmarking.rs

```diff
@@ -32,7 +32,7 @@ const USER_SEED: u32 = 999666;
 
 benchmarks! {
     submit_challenge_prove {
-        let v in 0 .. 20;
+        let v in 0 .. 20; // why 20? it is set to 100 in the pallet. Use constants
```

Comment:
OK, I have configured a constant for it:
https://github.com/CESSProject/cess/pull/88/files#diff-d3552cbf9043c7744c4c1073ae36fb86f92561afa8cf5a6ce99929f010619854R35